app/controllers/candidate_interface/personal_details_controller.rb

```diff
@@ -1,14 +1,14 @@
 module CandidateInterface
   class PersonalDetailsController < CandidateInterfaceController
     def edit
-      @personal_details_form = PersonalDetailsForm.new
+      @personal_details_form = PersonalDetailsForm.load(current_candidate.current_application)
```

Comment:
not sure about the name, in practice, it does not perform a *Load*, it is more a `build` IMO.

Reply:
We were unsure of `load` too, I'm happy with `build`!